src/ground-truth.rkt

```diff
@@ -74,12 +76,15 @@
       ([(k v) (in-hash t2)])
     (hash-set t1 k (+ (hash-ref t1 k 0) (* (/ v t2-total) t1-base)))))
 
-(define (sample-points precondition progs ctx)
+(define (sample-points precondition progs ctxlist)
   (timeline-event! 'analyze)
-  (define fn (make-search-func precondition progs ctx))
+  (define fn (make-search-func precondition progs ctxlist))
   (match-define (cons sampler table)
     (parameterize ([ground-truth-require-convergence #f])
-      (make-sampler ctx precondition progs fn)))
+      ;; TODO: Should make-sampler allow multiple contexts?
```

Comment:
I've thought about this and I think it probably should, so maybe rephrase the comment in the affirmative

Reply:
Could you rephrase what you mean by this?